addons/web/static/src/views/fields/properties/properties_field.xml

```diff
@@ -11,24 +11,24 @@
                     t-att-property-name="propertiesListGroup.name || ''">
                     <div t-if="!propertiesListGroup.invisibleLabel"
                         class="o_field_property_label o_field_property_group_label d-flex flex-row w-100 mb-3 text-uppercase fw-bolder align-items-baseline pe-2"
-                        t-att-class="{'invisible': propertiesListGroup.columnSeparator}"
+                        t-att-class="{'invisible': propertiesListGroup.columnSeparator, 'folded': isFolded}"
                         t-on-click="() => this.onSeparatorClick(propertiesListGroup.name)">
-                        <span t-if="propertiesListGroup.title" t-out="propertiesListGroup.title"/>
+                        <div t-if="foldable">
+                            <i class="fa fa-fw" t-att-class="isFolded ? 'fa-caret-right' : 'fa-caret-down'"/>
+                        </div>
+                        <span t-if="propertiesListGroup.title" t-out="propertiesListGroup.title" class="ps-2"/>
```

Comment:
Do we always want the "ps-2", even when not foldable? 

Maybe it's always foldable when there is a group title ^^